### PR TITLE
get all articles

### PR DESCRIPTION
**Pull request by Krishnainani**
This pull request will get you all the articles on the path of the articles.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+import Description from "./components/Description";
+import Header from "./components/Header";
+import Nav from "./components/Nav";
+
+const articleContext = createContext();
 
 function App() {
+  const [articles, setArticles] = useState([]);
   return (
+      <articleContext.Provider value={{ articles, setArticles }}>
```
Comment (douglashellowell) [inline src/App.js]: React Context is a very useful tool to share state in a component tree but i don't think it's appropriate in this case

At the moment when `setArticles` is called your *whole app* will re-render

You should try to hold your state as low down as possible in your component tree.

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    });
+  }, []);
+
+  return (
+    <div>
+      <h2>Articles :</h2>
+      {articles.map((article) => {
+        let date = Date(`${article.created_at}`);
+        return (
+          <li className="cards">
```
Comment (douglashellowell) [inline src/components/Articles.jsx]: you should always have a `ul` or an `ol` as the parent of a `li` element

File: src/components/Description.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ];
+const lengthOfQuotes = quotes.length;
+  const randomNum = Math.floor(lengthOfQuotes * Math.random());
+
+  return (
+    <div>
+      <q>{quotes[randomNum]}</q>
+    </div>
+  );
+}
```
Comment (douglashellowell) [inline src/components/Description.jsx:24]: Love this! 🧠

**Comment by douglashellowell** [comment]
great work!

With a slow connection it takes time for your articles to appear and users with poor signal only see a blank screen while this happens!
Think about what you want your users to see while the data is loading
You might need to check the conditional rendering notes for this 📖

(this goes for all api requests throughout your app)

File: src/App.js (hunk 1 above)
Comment (Krishnainani) [reply src/App.js]: Actually, according to my components diagram, I need to access the article's state on different levels of component so I thought of using context in the beginning, should I access it through props for now?

**Pull request merged**